app/components/employeeCard.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import styles from "./employeeCard.module.scss";
+import Link from "next/link";
+
+export default function EmployeeCard(props: {
+  employeeName: string;
+  employeeRole: string;
+  employeeImageSrc: string;
+  href: string;
+}) {
+  return (
+    <Link className={styles["link"]} href={props.href}>
+      <li className={styles["employeeCard"]}>
```

Comment:
could you make sure the `<li />` wrapps the `<Link />` for proper html syntax?

Reply:
Side note and perhaps not needed. Would it be beneficial for a11y if the links had an accessible label to them? In the rotor menu each name would be listed as links, which I suppose would give me as a user enough info to understand that each link would lead to more info of each name, but do we want to include what type of info, or what action is indicated? 

Also does it lead to a linkedIn Profile or a new page? Or does it open a modal??

Reply:
It redirects to a new page